src/30_Advanced/Image_Galleries_with_amp-carousel.html

```diff
@@ -142,9 +142,12 @@
 
   Whenever the carousel slide changes, it updates `selectedSlide`. This causes the `text`
   attribute in the `<p>` tag above the carousel to update as well.
+
+  Only [whitelisted functions](https://www.ampproject.org/es/docs/reference/components/amp-bind)
+  are allowed when using `amp-bind`. We use the syntax `+ selectedSlide` to make sure the variable is casted into an integer.
```

Comment:
s/We use the syntax `+ selectedSlide` to make sure the variable is casted into an integer./We use the unary plus operator before `selectedSlide` to cast it into an integer./

One more question: why are there parentheses around the expression?